src/Storefront/Controller/SearchController.php

```diff
@@ -43,32 +43,18 @@ class SearchController extends StorefrontController
     /** @var FindologicSearchService */
     private $findologicSearchService;
 
-    /**
```

Comment:
There are still a few imports in this class, which haven't been removed. Please remove them.